include/event_manager.h

```diff
@@ -296,6 +255,31 @@ void _event_submit(struct event_header *eh);
 int event_manager_init(void);
 
 
+/** Empty function for tracing event execution. Should be overloaded for profiling purposes. 
```

Comment:
Style note:
```
/** Brief description. More details.
```
This is javadoc brief (see https://www.doxygen.nl/manual/docblocks.html). Even when we use it please move the detail description to a separate line. It's easier to read. Should also match style.
```
/** Short, single-sentenced header ending with a dot.
  * More detailed description.
  **/
```

Reply:
As for the text - we cannot state that function is empty as it depends. Since it will be exported to upstream we cannot mention our profiler support. Maybe something like that.

```
/** Trace even execution.
  * The behavior of this function depends on the actual implementation. 
  * The default implementation of this function is no-operation. It is annotated as weak and is meant to be overloaded by layer adding support for profiling mechanism.
  **/
```

Reply:
Please add similar note to other functions.

Reply:
Pinging @greg-fer so he can check the wording and my style comment.